.github/workflows/build.image.yml

```diff
@@ -0,0 +1,54 @@
+name: Build image
+
+on:
+    push:
+        branches:
+            - main
+        paths:
+            - 'requirements/*'
```

Comment:
Don't forget to add the `docker/dockerfile` :wink:

Reply:
Good catch!